### PR TITLE
[ZRH-2024] go-live of program

### PR DESCRIPTION
**Pull request by tweinmann**
This is the final program for DoD Zurich 2024

### CODE REVIEW

File: data/events/2024-zurich.yml [hunk 1]
```diff
@@ -193,3 +194,250 @@ sponsor_levels:
...
+- title: Opening Afternoon
+  type: custom
+  date: 2024-04-16
+  start_time: '13:45'
+  end_time: '13:50'
+
+- title: "Ignites"
+  type: ignite
+  date: 2024-04-16
+  start_time: '13:50'
```
Comment (glasnt) [inline data/events/2024-zurich.yml:263]: You have 8 ignite speakers (presuming 5 minutes each) for a total of 40 minutes in a 30 minute slot. Is this intended? (Your day 2 has 35 minutes for 40 minutes of talks)

Comment (tweinmann) [reply data/events/2024-zurich.yml:263]: This is just the preliminary program. We'll change this going forward.

Comment (phrawzty) [reply data/events/2024-zurich.yml:263]: @tweinmann Thanks to the magic of Netlify, you have the preview here: https://deploy-preview-13615--devopsdays-web.netlify.app/

As this is clearly still a draft, please use your preview view as a visualisation aid. When you're ready to push to production, update this PR and we'll move forward. 👍

**Comment by tweinmann** [comment]
@glasnt @phrawzty we have finalized the program. please merge :)

**Review by phrawzty — APPROVED** [review]

**Pull request merged**